app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -746,6 +748,10 @@ class ThreadFragment : Fragment() {
             )
         }
 
+        getBackNavigationResult(OPEN_REACTION_BOTTOM_SHEET) { messageUid: String ->
+            navigateToEmojiPicker(messageUid, if (twoPaneViewModel.isThreadOpen) Thread else ThreadList)
```

Comment:
Is this condition truly correct on a tablet?

Reply:
Ok now I remember why it's correct but it's not intuitive. I feel like we should add a comment to make things clearer or rename the enum values. We can talk about it